blender_addons/io_md5_exporter/io_export_md5.py

```diff
@@ -1263,7 +1263,7 @@ def invoke(self, context, event):
 class ExportZipFileForTerasology(bpy.types.Operator):
   '''Create a ready to use module(.jar) that can be directly used in Terasology'''
   bl_idname = "export.zipfile"
-  bl_label =  "Export module"

```

Comment:
I changed it to "Export zip" as that is what appears on the window of the dialog.